### ISSUE TITLE
Support this project

### ISSUE THREAD

**Issue by Jawnnypoo** [ekx/GitLabAndroid#42]
@ekx Please consider merging in pull requests or turning over the app to someone who can manage it at this time. It is sad to let a project like this die, as it is a great foundation and has users on the Play Store.

**Comment by robinpaulson** [comment]
Yes, I'd like to see this happen too. @Jawnnypoo , @burrito82 , @JosephTurner and others have fixed a number of the issues, perhaps one of those contributors would be a good candidate if @ekx does not have time or inclination?

**Comment by burrito82** [comment]
I suggest that @ekx adds someone as an additional developer with commit/merge rights. This is if he would continue to publish versions to the Play Store. Otherwise, we could just establish a fork as the new main project, but distribution would be a lot more complicated.

**Comment by Jawnnypoo** [comment]
I think the fork might have to be the option as far as I can tell, I have tried emailing the developer about 3 times now. Looks like he is not managing the app on the Play Store either, since he has yet to give it a rating.

**Comment by telkomsel13** [comment]
#1

**Comment by robinpaulson** [comment]
It's been over a month now folks, I think it's acceptable to assume @ekx won't be continuing development of this. From a cursory glance at the "network" (https://github.com/ekx/GitLabAndroid/network) page, @Jawnnypoo looks like he has made the furthest progress in fixing problems, might I tentatively suggest using that fork as the one to base development efforts on? There are a few other patches from other forks which it would be useful to merge in as well. Any suggestions, comments, problems?

**Comment by Jawnnypoo** [comment]
I agree, and I have forked the repo. I have merged most of the pull requests in this repo:

https://github.com/Commit451/GitLabAndroid

**Comment by robinpaulson** [comment]
Fantastic, good to see. Now, is there any way to import the issues from the ekx repo to this?

**Comment by Jawnnypoo** [comment]
I have replicated most of the pull requests that are on this project, but I have not transferred the issues yet. I will get to that today. I also plan on releasing to the app store soon so that the app can gain traction so that others are not downloading the unmaintained version.

**Comment by burrito82** [comment]
Great news! :+1:

**Comment by Jawnnypoo** [comment]
It is worth mentioning on this issue, in case anyone stumbles across it, that the fork repo has been moved here:
https://gitlab.com/Commit451/GitLabAndroid

**Comment by robinpaulson** [comment]
And that fork has now been included in f-droid also.
